# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rustc/hotfix-x86-simd.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rustc-1.87.0-src/vendor/blake3-1.8.2/build.rs.vanilla	2025-05-29 16:24:12.122215659 +0200
+++ rustc-1.87.0-src/vendor/blake3-1.8.2/build.rs	2025-05-29 16:25:11.072177184 +0200
@@ -45,7 +45,7 @@
 
 fn is_x86_32() -> bool {
     let arch = &target_components()[0];
-    arch == "i386" || arch == "i586" || arch == "i686"
+    arch == "i386" || arch == "i486" || arch == "i586" || arch == "i686"
 }
 
 fn is_arm() -> bool {
